task_DEV_1/task_DEV_1/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace task_DEV_1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+          Console.WriteLine("Enter A number");
+           int inputNumber = int.Parse(Console.ReadLine());
```

Comment:
Wrong indents.

Reply:
https://github.com/ArteomKhromov/Homework/pull/1/files#diff-f7f86c4408232adb357cec8b2f59dfd416f724a80025d71500d1f6b7b52db25bR10-R27
Whole this part.